src/components/Scaler/index.tsx

```diff
@@ -172,11 +176,11 @@ class Scaler extends React.Component<Props, State> {
           x:
             item.xDomain ||
             stripPlaceholderDomain(Axes.x(oldDomainsByItemId[item.id])) ||
-            PLACEHOLDER_DOMAIN,
+            placeholder(Number.MIN_SAFE_INTEGER, Number.MAX_SAFE_INTEGER),
```

Comment:
I'll add some comments here explaining that. In short, `[0, 0]` was a hack that we can fix with the `placeholder` property. We want to set domains to be huge and subdomains to be small so that they still behave as you'd expect, though.